### PR TITLE
Adds Advanced Potion Merchant NPC

### PR DESCRIPTION
**Pull request by aleos89**
* **Addressed Issue(s)**: N/A

* **Server Mode**: Renewal

* **Description of Pull Request**: 
  * Adds the Advanced Potion Merchant who sells the Red, Yellow, White, and Blue Syrups for Concentrated Red Syrup Potion, Concentrated Blue Syrup Potion, and Concentrated Golden Syrup Potion creation.

### CODE REVIEW

**Review by Latiosu — APPROVED** [review]

**Review by Atemo — COMMENTED** [review]
The coordinates are taken from playing?

**Comment by attackjom** [comment]
iirc there are co-ordinate of this NPC in navigation files.

File: npc/re/merchants/shops.txt [hunk 1]
```diff
@@ -26,11 +26,13 @@
 // Alberta
 //=======================================================
 alberta_in,176,81,3	shop	Trading Merchant#alb	900,13200:-1,13221:-1,13222:-1,13150:-1,13102:-1,13151:-1,13154:-1,13155:-1,13163:-1,13165:-1,13168:-1
+alberta_in,176,97,3	shop	Advanced Potion Merchant#alb	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
 
 //=======================================================
 // Al De Baran
 //=======================================================
 alde_alche,38,184,3	shop	Material Seller#alche	755,7143:-1,7141:-1,7140:-1,6248:-1,6250:-1,6251:-1,6255:-1,6261:-1,6262:-1,6297:-1
+aldeba_in,99,55,3	shop	Advanced Potion Merchant#alde	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
```
Comment (Atemo) [inline npc/re/merchants/shops.txt:35]:
```suggestion
aldeba_in,99,56,5	shop	Advanced Potion Merchant#alde	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
```

File: npc/re/merchants/shops.txt [hunk 2]
```diff
@@ -26,11 +26,13 @@
 // Alberta
 //=======================================================
 alberta_in,176,81,3	shop	Trading Merchant#alb	900,13200:-1,13221:-1,13222:-1,13150:-1,13102:-1,13151:-1,13154:-1,13155:-1,13163:-1,13165:-1,13168:-1
+alberta_in,176,97,3	shop	Advanced Potion Merchant#alb	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
```
Comment (Atemo) [inline npc/re/merchants/shops.txt:29]:
```suggestion
alberta_in,176,97,5	shop	Advanced Potion Merchant#alb	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
```

File: npc/re/merchants/shops.txt [hunk 3]
```diff
@@ -58,6 +60,11 @@ que_ng,180,79,3	shop	Johnny Waiker	900,13200:-1,13221:-1,13222:-1
 dic_in01,238,107,5	shop	Peddler#dic	900,601:-1,602:-1,611:-1,610:-1
 dicastes01,207,200,6	shop	Points Merchant#dic	66,6360:-1,6361:-1,6362:-1,6363:-1
 
+//=======================================================
+// Geffen
+//=======================================================
+geffen_in,78,169,5	shop	Advanced Potion Merchant#gef	1_F_03,11621:-1,11622:-1,11623:-1,11624:-1
```
Comment (Atemo) [inline npc/re/merchants/shops.txt:66]:
```suggestion
geffen_in,78,169,3	shop	Advanced Potion Merchant#gef	1_F_03,11621:-1,11622:-1,11623:-1,11624:-1
```

File: npc/re/merchants/shops.txt [hunk 4]
```diff
@@ -79,6 +86,7 @@ izlude_d,128,158,7	duplicate(Vendor from Milk Ranch#i)	Vendor from Milk Ranch#d
 izlude_in,72,98,3	shop	Pet Groomer#iz	124,537:-1,643:-1,10013:-1,10014:-1,554:-1,6113:-1,6114:-1,6115:-1
 izlude_in,57,110,0	shop	Tool Dealer#iz	47,611:-1,501:-1,502:-1,503:-1,504:-1,506:-1,645:-1,656:-1,601:-1,602:-1,1065:-1,1750:-1
 izlude_in,72,102,3	shop	Trading Merchant#iz	900,13200:-1,13221:-1,13222:-1,13150:-1,13102:-1,13151:-1,13154:-1,13155:-1,13163:-1,13165:-1,13168:-1
+izlude_in,59,113,3	shop	Advanced Potion Merchant#iz	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
```
Comment (Atemo) [inline npc/re/merchants/shops.txt:89]:
```suggestion
izlude_in,59,113,5	shop	Advanced Potion Merchant#iz	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
```

File: npc/re/merchants/shops.txt [hunk 5]
```diff
@@ -121,6 +129,7 @@ morocc,166,54,2	shop	Jeweler#moc4	102,721:-1,723:-1,726:-1,728:-1,729:-1
 morocc,34,68,0	shop	Trader#moc7	93,748:-1
 morocc,269,193,4	shop	Trader#moc8	89,2609:-1,1516:-1,1522:-1
 morocc,256,191,5	shop	Trader#moc9	93,2612:-1
+morocc,170,84,7	shop	Advanced Potion Merchant#moc	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
```
Comment (Atemo) [inline npc/re/merchants/shops.txt:132]:
```suggestion
morocc,149,245,7	shop	Advanced Potion Merchant#moc1	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
morocc,170,84,1	shop	Advanced Potion Merchant#moc2	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
```

File: npc/re/merchants/shops.txt [hunk 6]
```diff
@@ -139,6 +148,7 @@ prontera,178,244,3	shop	Amatsu Trader#nin	83,13250:-1,13251:-1,13252:-1,13253:-1
 // Payon
 //=======================================================
 payon,123,109,4	shop	Trap Specialist#pay	66,7940:-1,12341:-1
+payon,157,96,5	shop	Advanced Potion Merchant#pay	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
```
Comment (Atemo) [inline npc/re/merchants/shops.txt:151]:
```suggestion
payon,157,96,3	shop	Advanced Potion Merchant#pay	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
payon_in01,12,53,5	shop	Advanced Potion Merchant#pay2	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
payon_in02,85,38,3	shop	Advanced Potion Merchant#pay3	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
```

File: npc/re/merchants/shops.txt [hunk 7]
```diff
@@ -157,6 +167,7 @@ prt_in,175,137,4	shop	Black Marketeer#prt	49,2139:-1,2800:-1,2801:-1,2802:-1,280
 s_atelier,17,110,1	shop	Part-Timer#sc_prt	67,6123:-1,6120:-1
 prontera,96,209,4	shop	Rebellion Accessories	564,25187:-1,7663:-1,7664:-1,7665:-1,7940:300,13200:-1,13221:-1,13222:-1,13215:-1,13216:-1,13217:-1,13218:-1,13219:-1,13220:-1,13228:-1,13229:-1,13231:-1,13232:-1,13230:-1
 prontera,92,209,4	shop	Rebellion Weapons	564,13120:-1,13122:-1,13189:-1,13195:-1,13192:-1,13193:-1,13194:-1,13197:-1,13198:-1,28200:-1,28201:-1
+prt_in,128,69,5	shop	Advanced Potion Merchant#prt	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
```
Comment (Atemo) [inline npc/re/merchants/shops.txt:170]:
```suggestion
prt_in,129,68,3	shop	Advanced Potion Merchant#prt	4_F_ALCHE_A,11621:-1,11622:-1,11623:-1,11624:-1
```

**Review by Atemo — CHANGES_REQUESTED** [review]
Ref for the npcs directions/coordinates https://github.com/rathena/rathena/issues/3681#issuecomment-459234254
I also checked the npcs coordinates on kro navi (april 2020)

**Comment by aleos89** [comment]
Ah, I didn't realize we had an issue open for them. I was running around cities on kRO to find them.

**Review by Atemo — APPROVED** [review]

**Pull request merged**